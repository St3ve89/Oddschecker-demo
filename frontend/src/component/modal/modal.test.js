import { Modal } from '.';
import renderer from 'react-test-renderer';

test('renders Modal component', () => {
  const modal = renderer.create(<Modal />).toJSON();
  expect(modal).toMatchSnapshot();
});
