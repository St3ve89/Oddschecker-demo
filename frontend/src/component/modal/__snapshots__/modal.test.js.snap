// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Modal component 1`] = `null`;
